Wrappers/Python/ccpi/optimisation/operators/GradientOperator.py

```diff
@@ -86,7 +86,7 @@ def domain_geometry(self):
     def range_geometry(self):
         return self.gm_range
                                    
-    def norm(self):
+    def calculate_norm(self):
```

Comment:
Why the need for a dedicated `calculate_norm` for `GradientOperator`? Wouldn't the standard inherited one work?

Reply:
Same as above